indico/modules/rb_new/client/js/modules/admin/SettingsPage.jsx

```diff
@@ -170,7 +170,51 @@ class SettingsPage extends React.PureComponent {
                                                max="30"
                                                required
                                                parse={value => (value ? +value : null)}
-                                               label={Translate.string('Monthly')}
+                                               label={Translate.string('Monthly bookings')}
+                                               validate={v.range(1, 30)}
+                                               disabled={fprops.submitting} />
+                                    </Form.Group>
+                                </Message>
+                            </FieldCondition>
+                            <Field name="end_notifications_enabled" component={ReduxCheckboxField}
+                                   componentLabel={Translate.string('Send reminders when bookings are about to end')}
+                                   disabled={fprops.submitting} />
```

Comment:
let's a add a description here for consistency with the other checkbox

Reply:
I didn't add it there since, at least to me, the label is pretty explanatory.

Reply:
then let's remove it from the other checkbox

Reply:
fair enough ;)